tabs/tab_corr.py

```
@@ -1,300 +1,342 @@
+                )
             )
         )
     )
 
 
-def corr_server(namespace: str, df: reactive.Value, var_meta: reactive.Value, 
-                df_matched: reactive.Value, is_matched: reactive.Value):
+# ✅ Use @module.server decorator properly
+@module.server
+def corr_server(
+    input: Any, 
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `input`

(ANN401)